src/components/Icon/Icon.enums.ts

```diff
@@ -10,6 +10,7 @@ export const SSCIconNames = {
   quoteSquare: 'quote-square',
   caretDown: 'caret-down',
   plus: 'plus',
+  search: 'search',
```

Comment:
Please rebase on top of `datatable` branch. This file was moved to `theme/icons/icons.enum.ts`